### PR TITLE
Switch to Twemoji

### PR DESCRIPTION
**Pull request by TheEssemCraft**
Here it is, the port to Twemoji that I've been working on. If you notice some extra stuff that's unneeded, it's probably because [the SVGs used to be very overcomplicated](https://imgur.com/a/1xWsL). I know you might have some questions, so here are some answers:

- **Q: Why did you change the `execMaxBuffer` in `Gruntfile.js`?**
A: During my testing, I found that the maxBuffer kept overflowing, so this was necessary to keep it from doing so.
- **Q: Why did you change the color value for green in `layerize.js`? Why did you add navy as a color?**
A: I changed the value for green to more closely align with the actual SVG color value. Also, I added navy because some SVGs use that color name.
- **Q: Why did you replace `resemble.outputSettings` in the test suite JavaScript?**
A: In version 3.0.0 of Resemble.js, outputSettings will be removed, so I had to change it for compatibility with future versions.
- **Q: Why are there so many commits?**
A: Since I use multiple computers and I don't have a credit/debit card, I treat GitHub more like a workspace. Due to this, many commits are either debugging or testing stuff that either did or didn't work out. I used Travis CI to test the build process and Heroku for the test suite.

If you have any more questions, feel free to ask.

### CODE REVIEW

**Comment by timdream** [comment]
@TheEssemCraft Hi! Thank you so much for working on this.

Don't worry about the extra commits — I can squash it when we merge. I am kicking off the internal conversation again just to make sure Twemoji is something we want. If so I will help you get this merged and check-in to the Firefox source code.

Noted that given newer OSes come with their own Emoji font, eventually, we will want to stop bundling our own Emoji font in Firefox.

File: LICENSE.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ## License for the Code
 
-Copyright 2016, Mozilla Foundation
+Copyright 2018, Mozilla Foundation
```
Comment (timdream) [inline LICENSE.md]: Should be `2016-2018`

File: package.json [hunk 2]
```diff
@@ -1,39 +1,40 @@
...
-    "grunt-webfont": "^1.4.0",
-    "load-grunt-tasks": "^3.5.0",
+    "grunt-webfont": "^1.6.0",
+    "load-grunt-tasks": "^3.5.2",
     "rmdir": "^1.2.0",
     "unzip": "^0.1.11",
-    "xml2js": "^0.4.16",
-    "xmlbuilder": "^8.2.2"
+    "xml2js": "^0.4.19",
+    "xmlbuilder": "^8.2.2",
```
Comment (timdream) [inline package.json]: Remove this comma.

File: extras/1f3fb.svg [hunk 3]
```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 64 64" enable-background="new 0 0 64 64"><circle fill="#ffe1bd" cx="32" cy="32" r="30"/></svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 36 36"><path fill="#F7DECE" d="M33 30c0 1.657-1.343 3-3 3H6c-1.657 0-3-1.343-3-3V6c0-1.657 1.343-3 3-3h24c1.657 0 3 1.343 3 3v24z"/></svg>
```
Comment (timdream) [inline extras/1f3fb.svg]: Any reason that you keep these `svg` files in `extras`? They came from Twemoji and exists in your zip file.

Comment (TheEssemCraft) [reply extras/1f3fb.svg]: I thought they were part of EmojiOne, since I didn't see that part of `LICENSE.md` originally.

**Comment by timdream** [comment]
@TheEssemCraft Thanks for the changes! I will take it from here.

**Pull request merged**